src/App/App.xaml.cs

```diff
@@ -164,6 +167,39 @@ public App(AppOptions appOptions)
             });
         }
 
+        private async Task SyncPasswordlessLoginRequestsAsync()
+        {
+            var activeUserId = await _stateService.GetActiveUserIdAsync();
+            // if the user has not enabled passwordless logins ignore requests
+            if (!await _stateService.GetApprovePasswordlessLoginsAsync(activeUserId))
+            {
+                return;
+            }
+
+            var loginRequests = await _authService.GetPasswordlessLoginRequestsAsync();
+            if (loginRequests == null || !loginRequests.Any())
+            {
+                return;
+            }
+
+            var validLoginRequests = loginRequests.Where(l => l.RequestApproved == null && l.ResponseDate == null
+                && l.CreationDate.ToUniversalTime().AddMinutes(Constants.PasswordlessNotificationTimeoutInMinutes) > DateTime.UtcNow).ToList();
+
+            if (validLoginRequests == null || !validLoginRequests.Any())
+            {
+                return;
+            }
+
+            var validLoginRequest = validLoginRequests.OrderByDescending(x => x.CreationDate).First();
```

Comment:
This can be written as:
```c#
var validLoginRequest = loginRequests.Where(l => l.RequestApproved == null 
                                                 && l.ResponseDate == null
                                                 && l.CreationDate.ToUniversalTime().AddMinutes(Constants.PasswordlessNotificationTimeoutInMinutes) > DateTime.UtcNow)
                                     .OrderByDescending(x => x.CreationDate)
                                     .FirstOrDefault();

if (validLoginRequest is null)
{
    return;
}
```